### PR TITLE
Update Input Fields look

### LINKED ISSUE #1742: Update Input Fields look [Fiji]
This is almost the last update of Fiji. The one remaining is a redesign of ERC20, ERC721 tokens. 

Zeplin:

1. Blank
https://app.zeplin.io/project/5d088205bff2d15de6a4397b/styleguide/components?cseid=5e218a805b5b617d8eec7aef

2. Entering
https://app.zeplin.io/project/5d088205bff2d15de6a4397b/styleguide/components?cseid=5e218c6773e393a79c640116

3. Entered
https://app.zeplin.io/project/5d088205bff2d15de6a4397b/styleguide/components?cseid=5e218cbacf6a85ab00ee4d3b

4. Error
https://app.zeplin.io/project/5d088205bff2d15de6a4397b/styleguide/components?cseid=5e218d1b0f5ab5ab7cdca668

----

input_fields_ios

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #1742

### CODE REVIEW

**Comment by hboon** [comment]
Can you fix the conflict? Maybe it's because master has changed Podfile and hence Podfile.lock since you pulled from master.

**Comment by hboon** [comment]
Read my comments in #1815 and #1817 about the git commands first before you look at this:

There are merge commits in this PR. Do this:

```
git fetch origin master:master #gets the updated `master` and updates it locally
git rebase master
```

It tries to rebase master and stops, complaining that it can't merge Podfile.lock including these error messages:

```
#...
CONFLICT (content): Merge conflict in Podfile.lock
Auto-merging Podfile
Auto-merging AlphaWallet/Transfer/ViewControllers/SendViewController.swift
Removing AlphaWallet/Assets.xcassets/qr_code_icon.imageset/qr_code_icon.png
Auto-merging AlphaWallet.xcodeproj/project.pbxproj
error: Failed to merge in the changes.
#...
```

The rebase operation is still running, waiting for us to fix it.

Check where are the conflicts:

```
git status -s
```

```
M  AlphaWallet/Wallet/ViewControllers/ImportWalletViewController.swift
M  Podfile
UU Podfile.lock
```

So it's `Podfile.lock`. Since it's tightly related to `Podfile`, we should verify that file is good too. This conflict is because master has and updated `Podfile` and `Podfile.lock` and we have changed it in this PR too.

Since it's `Podfile.lock`, it's a simple fix. Just remove the changes introduced in `Podfile.lock` by this PR which is causing the conflict (git will have updated the file with <<< and >>> markers) and and:

```
$ pod install #We have to remove the <<< and >>> confilicts in Podfile.lock first so the cocoapod tool can help us update it
$ git add Podfile.lock
$ git rebase --continue #resume the suspended git rebase master operation

Applying: Update ui for AmountTextField, AddressTextField, update text fields and text views, on add custom token, import tokens, send tokesn screens.
Applying: Update textfields on EditMyDappViewController screen
Applying: Fixed:    
```

Run:

```
$ git log

commit 83533e76c345c5eee79918283e7ec8adb2f31773 (HEAD -> TextFields_redesign)
Author: Vladyslav shepitko <vlad_shepitko@outlook.com>
Date:   2020-05-20 09:23:34 +0300

    Fixed:
    - paste button
    - clear button
    - add USA flag to amount textbox
    - swap address textbox and amount textboxes
    - update textfields in SliderTextFieldCell
    - remove addressBookButton

commit dad1d846cda06ed84c4985647724eda9a88ad171
Author: Vladyslav shepitko <vlad_shepitko@outlook.com>
Date:   2020-05-18 14:57:14 +0300

    Update textfields on EditMyDappViewController screen

commit b86a9d1f43b5db46ccd7c7d6adb8793b73fa4987
Author: Vladyslav shepitko <vlad_shepitko@outlook.com>
Date:   2020-05-18 13:11:38 +0300

    Update ui for AmountTextField, AddressTextField, update text fields and text views, on add custom token, import tokens, send tokesn screens.

commit 1e5a20734882675cf6a59fd96e558f2f42df36a7 (origin/master, master)
Merge: 3d52fb8e5 143670bed
Author: James Sangalli <j.l.sangalli@gmail.com>
Date:   2020-05-21 12:06:33 +1000

    Merge pull request #1800 from AlphaWallet/upgrade-schema-2020-06
    
    Upgrade schema 2020 06
```

Because of the rebase master, the commits in the feature branch are now over the tip of the master branch.

I think these set of commits is an atomic change "Update Input Fields look", so I'll just rewrite history and squash them into a single commit (didn't paste the commands for this here to avoid too much noise).

It becomes:

```
commit 5961f058d5330389f889998d2acf4b9957bc43b7 (HEAD -> TextFields_redesign)
Author: Vladyslav shepitko <vlad_shepitko@outlook.com>
Date:   2020-05-18 13:11:38 +0300

    [Fiji] Update Input Fields look #1742
    
    Including:
    
    * AmountTextField
    * AddressTextField
    * usage of UITextField

commit 1e5a20734882675cf6a59fd96e558f2f42df36a7 (origin/master, master)
Merge: 3d52fb8e5 143670bed
Author: James Sangalli <j.l.sangalli@gmail.com>
Date:   2020-05-21 12:06:33 +1000

    Merge pull request #1800 from AlphaWallet/upgrade-schema-2020-06
    
    Upgrade schema 2020 06
```

1 commit (sometimes multiple commits make sense too). I have updated the first to show the summary and end it with the issue number. We have tagged it with `[Fiji]` to reflect that it's part of a bigger thing. That's great. I moved that to the front:

```
[Fiji] Update Input Fields look #1742
```

Since your commits originally had more detail, I shorten them a bit (don't have to be too detailed unless it's a big change or if we are explaining the "why" which we can't figure out from the code) and added them too, ending up with the commit message:
    
```
[Fiji] Update Input Fields look #1742

Including:

* AmountTextField
* AddressTextField
* usage of UITextField
```

**Comment by hboon** [comment]
Will review. Thanks

File: AlphaWallet/UI/SelectCurrencyButton.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.setContentHuggingPriority(.required, for: .horizontal)
+        label.setContentCompressionResistancePriority(.required, for: .horizontal)
+        label.font = DataEntry.Font.amountTextField
+        
+        return label
+    }()
+    
+    private let expandImageView: UIImageView = {
```
Comment (hboon) [inline AlphaWallet/UI/SelectCurrencyButton.swift:22]: Can you hide this when toggling between fiat/token is not available?

There's a line like this in SendViewController:

```
amountTextField.alternativeAmountLabel.isHidden = true
```

Maybe both of them can be extracted into a new `isAlternativeAmountEnabled` property in `AmountTextField`?

**Comment by vladyslav-iosdev** [comment]
> WIP

seems like no, only few fixes, according you comments and that all.

File: AlphaWallet/UI/SelectCurrencyButton.swift (hunk 1 above)
Comment (vladyslav-iosdev) [reply AlphaWallet/UI/SelectCurrencyButton.swift:22]: > git fetch origin master:master #gets the updated `master` and updates it locally

when i do this there error shows. 
Refusing to fetch into current branch refs/heads/master of non-bare repository

Comment (hboon) [reply AlphaWallet/UI/SelectCurrencyButton.swift:22]: You can do it in a few steps:

git checkout master
git pull origin master #safe if you haven’t commit to master manually

Comment (hboon) [reply AlphaWallet/UI/SelectCurrencyButton.swift:22]: Remember that origin is probably “upstream” for you

Comment (vladyslav-iosdev) [reply AlphaWallet/UI/SelectCurrencyButton.swift:22]: > You can do it in a few steps:
> 
> git checkout master
> git pull origin master #safe if you haven’t commit to master manually

did it.
it tells next
 * branch              master     -> FETCH_HEAD
Already up to date.

Comment (vladyslav-iosdev) [reply AlphaWallet/UI/SelectCurrencyButton.swift:22]: seems like i can push my changes, right?

Comment (hboon) [reply AlphaWallet/UI/SelectCurrencyButton.swift:22]: Sure.

**Comment by hboon** [comment]
@vladyslav-iosdev is this ready for review?

**Comment by vladyslav-iosdev** [comment]
> @vladyslav-iosdev is this ready for review?

yes

**Pull request merged**